lib/vva/services/document_list.rb

```diff
@@ -17,22 +17,26 @@ def get_by_claim_number(claim_number)
       document_list = response.body[:get_document_list_response][:dcmnt_record]
 
       return [] if document_list.blank?
+      # Savon parses one item list as a Hash instead of an Array
+      document_list.is_a?(Hash) ? [create_record(document_list)] : document_list.map { |r| create_record(r) }
```

Comment:
This turns anything into an array, and is a bit more concise.

`[document_list].flatten.map { |r| create_record(r) }`

Reply:
This is def. more beautiful.